src/App.js

```diff
@@ -9,7 +10,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    {Article(article)}
```

Comment:
You don't need this line. just pass  'article' properties to the Article component as props.

Reply:
Fixed!